plugins/catalog/src/apis/EntityPresentationApi/DefaultEntityPresentationApi.ts

```diff
@@ -304,7 +304,13 @@ export class DefaultEntityPresentationApi implements EntityPresentationApi {
     };
   }
 
-  #getEntityForInitialRender(entityOrRef: Entity | string): {
+  #getEntityForInitialRender(
+    entityOrRef: Entity | string,
+    context?: {
+      defaultKind?: string;
+      defaultNamespace?: string;
+    },
```

Comment:
This is actually not right - the `entityOrRef` MUST be a full entity ref if it's a string, so there's no default kind or namespace to be had, because it must contain them both already.

Maybe the _caller_ of this method needed to use careful string parsing to ensure that it passed in an actual ref, instead of for example some shorthand out of the `spec`?

Reply:
Ok I will remove it 🙂 
I thought the presentation api was meant to handle the parsing also